network/radicale/radicale.SlackBuild

```diff
@@ -90,7 +90,7 @@ rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
 rm -rf $PRGNAM-$VERSION
-tar xvf $CWD/$PRGNAM-$VERSION.tar.gz || tar xvf $CWD/Radicale-$VERSION.tar.gz || tar xvf v$VERSION.tar.gz
+tar xvf $CWD/$PRGNAM-$VERSION.tar.gz
```

Comment:
The name of the project is actually `Kozea/Radicale` - the current link needs to change:

from https://github.com/SlackBuildsOrg/slackbuilds/blob/cdef13221b8fac6b940471ce1841accae367eb2d/network/radicale/radicale.info#L4C11-L4C91

It should be:

https://github.com/Kozea/Radicale/archive/refs/tags/v3.1.8/Radicale-3.1.8.tar.gz

Reply:
hmm not sure, lowercase version should work too

Reply:
also, I don't understand how it works, it says that radicale-3.1.8.tar.gz does not exist but then it builds the package?

Reply:
based on our SBo Github Guide, name of the tarball should match with the project name to give consistent output

Reply:
But is it related to the build failure? If I changed a link to https://github.com/Kozea/Radicale/archive/refs/tags/v3.1.8/Radicale-3.1.8.tar.gz I get:

    $ sudo env MAKEFLAGS='-j24 -l13' nice -n 5 bash ./radicale.SlackBuild
    tar: /home/ja/slackbuilds/network/radicale/radicale-3.1.8.tar.gz: Cannot open: No such file or directory
    tar: Error is not recoverable: exiting now

I want to know why did the build fail.

Reply:
so the build robot does not download files specified in .info files?

Reply:
I think that robot uses this for building https://github.com/aclemons/slackbuilds-ci/blob/master/.slackbuilds-library/resources/buildpkg#L1, I'll try to understand how it works.

Reply:
> I want to know why did the build fail.

The build is run and downloads the links from the info file with content-disposition - just like it you navigate to the link on the SBo website or open the link in your browser.

So if you take the link currently in the info file:

https://github.com/Kozea/Radicale/archive/refs/tags/v3.1.8/radicale-3.1.8.tar.gz

open it firefox or chromium, you get Radicale-3.1.8.tar.gz.

But you removed handling for that filename from the SlackBuild. That is why the build currently fails.

> also, I don't understand how it works, it says that radicale-3.1.8.tar.gz does not exist but then it builds the package?

But it does fail:

```
Running radicale.SlackBuild ...                                     ETA 21:16:??
env MAKEFLAGS='-j24 -l13' nice -n 5 bash ./radicale.SlackBuild
network/radicale: radicale.SlackBuild failed (status 2)                 21:16:06
tar: /tmp/SBo/repo.7jb9St/slackbuild_radicale/radicale-3.1.8.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
See /var/log/repo/SBo/network/radicale/rebuild.log
Unmounting chroot ... done.
⛔️ network/radicale FAILED ⛔️
```

The github link guide is there to make it so you get the same filename whether you download it this way with a browser or anything that respects the content-disposition, or if you directly wget it and have wget us the last URI part as the filename.

Reply:
ok, thx for explanation. I believe that the robot goes on to using https://github.com/aclemons/slackrepo, I just wanted to understand what it does.

Reply:
oh no, I'll run linters again...

Reply:
> ok, thx for explanation. I believe that the robot goes on to using https://github.com/aclemons/slackrepo, I just wanted to understand what it does.

Yes that's correct. The comments hit a github webhook which triggers a jenkins job which runs slackrepo to build the package. Jenkins then comments on this PR with the results.